TRIAGING.md

```diff
@@ -0,0 +1,118 @@
+
+- Do now
+- Do sometime
+- Provide user support
+- Never do (_i.e._, close)
+
+At its core, triaging enables new issues to be sorted into these four categories, which helps to ensure that they will be processed at a velocity similar to or exceeding the rate at which new issues are coming in. One of the benefits of actively triaging issues is to avoid engineer burnout and make necessary work sustainable; this is done by eliminating a never-ending backlog that has not been reviewed by any maintainers.
+
+There will always be broad-scope design and architecture implementations that the `conda` team will be interested in pursuing; by actively triaging issues, the triaging engineers will be able to more easily track and tackle both specific and big-picture goals.
+
+### Who triages?
+
+<!-- This section is still a WIP, there is no official triaging engineer team yet to link to -->
+
+Triaging engineers are a `conda` governance [sub-team](https://github.com/conda-incubator/governance#sub-teams); they are a group of Anaconda and community members who are responsible for making decisions regarding closing issues and setting feature work priorities, amongst other triage-related tasks. The list of current members is listed [here](future link)
+
+
+### How do items show up in triaging?
+
+New issues that are opened in any of the repositories in the [`conda` GitHub project](https://github.com/conda) will show up in the `New` column of the [Triaging board](https://github.com/orgs/conda/projects/4). This process is executed via [GitHub Actions](https://docs.github.com/en/actions). The two main GitHub Actions workflows utilized for this purpose are [Issues](https://github.com/conda/infra/blob/main/.github/workflows/issues.yml) and [Boards](https://github.com/conda/infra/blob/main/.github/workflows/boards.yml).
+
+The GitHub Actions in the `conda/infra` repository are viewed as canonical; the [Sync workflow](https://github.com/conda/infra/blob/main/.github/workflows/sync.yml) sends out any modifications to [other `conda` repositories](https://github.com/conda/infra/blob/main/.github/sync.yml#L3-L17) from there.
+
+
+### What is done about the issues in "triaging" mode?
+
+Issues in the "Triaging" column of the board have been reviewed by a triaging engineer and are considered ready for the following procedures:
+
+- Mitigation via short-term workarounds and fixes
+- Redirection to the correct project
+- Determining if support can be provided for errors and questions
+- Closing out of any duplicate/off-topic issues
+
+The triaging engineer on rotation is not seeking to _resolve_ issues that arise. Instead, the goal is to understand the ticket and to understand whether it is an issue in the first place and to then collect as much relevant information as possible so that the maintainers of `conda` can make an informed decision on the appropriate resolution schedule necessary.
```

Comment:
```suggestion
The triaging engineers on rotation are not seeking to _resolve_ issues that arise. Instead, the goal is to understand the ticket and to understand whether it is an issue in the first place and to then collect as much relevant information as possible so that the maintainers of `conda` can make an informed decision on the appropriate resolution schedule necessary.
```